mailbox/spring/src/main/resources/META-INF/spring/event-system.xml

```diff
@@ -20,14 +20,19 @@
 
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:util="http://www.springframework.org/schema/util"
        xsi:schemaLocation="
-          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+          http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
```

Comment:
I followed [this answer](https://stackoverflow.com/a/11540129/6072746)

Reply:
Also valid ^^